data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -46,13 +46,12 @@ then
 		"${release}" \
 		render \
 			--output-dir=/assets/cvo-bootstrap \
-			--release-image="${release}"
+			--release-image="${release}" &&
 
-	cp cvo-bootstrap/bootstrap/* bootstrap-manifests/
-	cp cvo-bootstrap/manifests/* manifests/
-	## FIXME: CVO should use `/etc/kubernetes/bootstrap-secrets/kubeconfig` instead
-	cp auth/kubeconfig /etc/kubernetes/kubeconfig
-fi
+	cp cvo-bootstrap/bootstrap/* bootstrap-manifests/ &&
+	cp cvo-bootstrap/manifests/* manifests/ &&
+	cp auth/kubeconfig /etc/kubernetes/kubeconfig # FIXME: CVO should use `/etc/kubernetes/bootstrap-secrets/kubeconfig` instead
```

Comment:
nit - my 0.02 $ if i may @sttts - will be nice to leave the FIXME comment above the line like it was before rather then having a long line ... although is all individual preference

Reply:
I would have bet that bash falls over this, with `&&` and comment. But it doesn't. Fixed.